### PR TITLE
Add survey form index.html & styles.css

### PR DESCRIPTION
**Pull request by emmanuelcortes2**
## Survey form

- Added index.html & Styles.css.
- Added user data inputs.
- Added 'role', 'recommendations' & 'favorite section' selectors.
- Added  'improvements' selectors.
- -Added comments text area.
- Added submit button
- Styled every element mentioned before.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+        <label  class="labs"
+          ><input
+            name="prefer"
+            value="additional-courses"
+            type="checkbox"
+            class="input-checkbox"
+          />Additional Courses</label
+        >
+        <br><br>
+      </div>
```
Comment (emmanuelcortes2) [inline index.html:172]: 1. The text needs to be aligned to the left in this section. Kindly, align the text to the left so it matches the original design.
2. Please add padding to the check boxes so they match the original design.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+          name="comment"
+          placeholder="Enter your comment here..."
+        ></textarea>
+      </div>
+      <br>
+      <div class="form-group">
+        <button type="submit" id="submit" class="submit-button">
+          Submit
+        </button>
+      </div>
```
Comment (emmanuelcortes2) [inline index.html:188]: Submit button needs to be the same width as the "comments & suggestion text area. Please change the width of the submit button so it matches the text area width.

**Review by emmanuelcortes2 — COMMENTED** [review]
# Changes required

- Please check all my comments below and make the required changes

## Optional

- Please add a font-family property that matches the original font design

**Review by Owusu-Desmond — APPROVED** [review]
Hi @emmanuelcortes2 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
📌 Good to use inline comments for your review.✅
📌 Good communication.
📌 Good to give a reason why you requested a changes

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**